doc/manuals/admin/cli.md

```diff
@@ -109,6 +109,8 @@ The list of available options is the following:
     broker process.
 -   **-httpTimeout <interval>**. Specifies the timeout in milliseconds
     for forwarding messages and for notifications.
+-   **-cprForwardLimit**. Maximum number of forwarded requests to Context Providers for a single client request
```

Comment:
Surprisingly, this parameter wasn't included in the documentation. I have take advantage of this PR to do so.

NTC